config/manifests/bases/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     capabilities: Seamless Upgrades
     containerImage: icr.io/cpopen/common-service-operator:latest
     createdAt: "2020-10-19T21:38:33Z"
-    description: The IBM Common Service Operator is used to deploy IBM Common Services
+    description: The IBM Cloud Pak foundational services operator is used to deploy IBM foundational services.
```

Comment:
same cmments as in the other location of CSV

Reply:
It is not a new line, it's all on line 9.